### PR TITLE
Enable subscription-controller monitoring

### PR DESCRIPTION
**Pull request by montaro**
See also #2602

### CODE REVIEW

File: resources/event-bus/charts/subscription-controller-knative/values.yaml [hunk 1]
```diff
@@ -5,3 +5,6 @@ port: 8080
 replicaCount: 1 # the subscription controller is deployed as one replica instance
 image:
   pullPolicy: IfNotPresent
+monitoring:
+  grafana: enabled
+  alerts: enabled
```
Comment (marcobebway) [inline resources/event-bus/charts/subscription-controller-knative/values.yaml]: Please add a newline here.

File: resources/event-bus/charts/subscription-controller-knative/templates/deployment.yaml [hunk 2]
```diff
@@ -15,8 +15,10 @@ spec:
       annotations:
         sidecar.istio.io/inject: "true"
       labels:
-        app: "event-bus-subscription-controller-knative" #This is changed because of tracing UI.
+        app: {{ template "subscription-controller-knative.fullname" . }} #This is changed because of tracing UI.
```
Comment (suleymanakbas91) [inline resources/event-bus/charts/subscription-controller-knative/templates/deployment.yaml]: You can delete the comment here

Comment (montaro) [reply resources/event-bus/charts/subscription-controller-knative/templates/deployment.yaml]: OK

**Pull request merged**